sympy/physics/quantum/tests/test_grover.py

```diff
@@ -70,16 +70,16 @@ def test_grover_iteration_2():
     # After (pi/4)sqrt(pow(2, n)), IntQubit(2) should have highest prob
     # In this case, after around pi times (3 or 4)
     iterated = grover_iteration(basis_states, v)
-    iterated = qapply(iterated)
+    iterated = qapply(iterated, mul=True) # mul=True indicates terms must be simplified by expansion
```

Comment:
We generally want to stick to a line limit of around 80 character per PEP 8 unless there is a reason to have a longer line. Here the comment can just be on a preceding line (and doesn't need to be repeated several times.

Reply:
1) I'll work on the line length, will take some time.
2) I tend to repeat comments as from my experience it helps getting the gist of some code just by looking at the spot. Stating things once won't help if someone is just trying to understand a code segment some 50 lines below.

Reply:
Have not yet fixed line length in the test file.

Reply:
line length fixed in test_qapply.py